app/src/main/java/com/epam/themes/backend/entities/Object.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.themes.backend.entities;
+
+public class Object extends BaseEntity {
```

Comment:
It's very bad idea to call some class 'Object', because In java all classes inherit implicitly from Object class